ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -324,6 +324,10 @@ <h3> Generated Resources:</h3>
         <span *ngIf="resourcesGenerated.DataflowJobName!=='' && (selectedMigrationType ==='lowdt') && !isSharded">
             <li>Dataflow job: <a [href]="resourcesGenerated.DataflowJobUrl"
                     target="_blank">{{resourcesGenerated.DataflowJobName}}</a>
+                    <span><button mat-button class="configure" (click)="openGcloudPopup(resourcesGenerated.DataflowGcloudCmd)">
+                        <mat-icon matTooltip="Equivalent gCloud command" matTooltipPosition="above">
+                            code</mat-icon>
+                    </button></span>
```

Comment:
For the screenshot, it doesn't look very obvious what the `< >` button would do...I am not sure what is the best UX experience here.
@bharadwaj-aditya @shreyakhajanchi thoughts?

Reply:
I have added a tool tip as well which on hover which might help, could'nt find a better icon.

<img width="290" alt="Screenshot 2023-09-26 at 3 33 26 PM" src="https://github.com/GoogleCloudPlatform/spanner-migration-tool/assets/19630018/5825182f-f7f2-44dc-b7fa-9b860ed3374e">